pom.xml

```diff
@@ -1,370 +1,280 @@
-                    <artifactId>jboss-javaee-6.0</artifactId>
-                    <type>pom</type>
-                </dependency>
-                <dependency>
-                    <groupId>org.jboss.weld</groupId>
-                    <artifactId>weld-core</artifactId>
-                    <version>${version.weld}</version>
-                    <scope>test</scope>
-                </dependency>
-                <dependency>
-                    <groupId>org.slf4j</groupId>
-                    <artifactId>slf4j-log4j12</artifactId>
-                    <version>${version.slf4j}</version>
-                    <scope>test</scope>
-                </dependency>
-                <dependency>
-                    <groupId>log4j</groupId>
-                    <artifactId>log4j</artifactId>
-                    <version>${version.log4j}</version>
-                    <scope>test</scope>
-                </dependency>
-            </dependencies>
-            <build>
-                <plugins>
-                    <plugin>
-                        <artifactId>maven-surefire-plugin</artifactId>
-                        <configuration>
-                            <excludes>
-                                <exclude>%regex[.*container.*]</exclude>
-                            </excludes>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-    </profiles>
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-release-plugin</artifactId>
+				<configuration>
+					<autoVersionSubmodules>true</autoVersionSubmodules>
+					<pushChanges>false</pushChanges>
+					<localCheckout>true</localCheckout>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<systemProperties>
+						<!-- <jboss.version>${version.wildfly}</jboss.version> -->
+						<path.tools_jar>${path.tools_jar}</path.tools_jar>
+					</systemProperties>
+					<runOrder>alphabetical</runOrder>
+					<excludes>
+						<exclude>%regex[.*local.*]</exclude>
+					</excludes>
+				</configuration>
+			</plugin>
+			<!-- We need to jarjar this, since WildFly grabs all org.jboss.byteman 
+				as system packages -->
+			<plugin>
+				<groupId>org.sonatype.plugins</groupId>
+				<artifactId>jarjar-maven-plugin</artifactId>
+				<version>${version.jarjar}</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>jarjar</goal>
+						</goals>
+						<configuration>
+							<includes>
+								<include>org.jboss.byteman:byteman-submit</include>
+							</includes>
+							<rules>
+								<rule>
+									<pattern>org.jboss.byteman.**</pattern>
+									<result>org.jboss.arquillian.extension.byteman.@1</result>
+								</rule>
+							</rules>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
+
+	<profiles>
+		<profile>
```

Comment:
I'm not really sure if we need those profiles. We can simply run the build as follows:

```
$ mvn clean install -Darq.container.chameleon.configuration.chameleonTarget="jboss as:7.1.1.Final:managed"
```